lib/pkt-line.js

```diff
@@ -90,7 +91,7 @@ function deframer(emit) {
         throw new Error("pkt-line decoder in invalid state");
       }
     }
-
+    if (doUndefined) emit();
```

Comment:
We want to emit undefined, but only after the http request that pulled in the delta's. the above two lines achieve this for a clean clone. I'm not sure this will also work for a fetch. I haven't read the documentation on the smart http protocol yet.